content/2.developers/3.guides/9.cw-fees/2.gas-guzzler.md

```diff
@@ -0,0 +1,148 @@
+---
+
+objectID: developers_guides_fee-grant_guzzler-gasless-transactions
+title: Gasless transactions via Gas Guzzler
+description: How to enable gasless transactions in your dapps by utulizing the Guzzler Club contracts
+parentSection: Developers
+parentSectionPath: /developers
+---
+
+#  Guzzler club gasless transaction
+
+The Guzzler Club offers a gasless transaction solution on the Archway network, allowing users to interact with decentralized applications (dapps) without needing an ARCH token balance in their wallet. This guide covers how the solution works, core features, and how users can benefit from gasless transactions. You can access the Guzzler Club [here](https://connect.archway.io/guzzler-club).
+
+## Benefits to users
+
+1. **New Archway Explorers**: Enable new users to use dapps on Archway without requiring an initial ARCH balance.
+2. **Archway Pros**: Allow experienced users to save on transaction costs by leveraging free transactions.
+
+## Core features
+
+1. **Finite Allocation of ARCH for Gas**: 
+  - Every account receives a finite allocation of ARCH to spend on gas per epoch (21 days).
+  - This allocation is automatically renewed at the start of each epoch.
+
+2. **Earn Additional Free Transactions**:
+   - Users can increase their allocation of free transactions by completing basic tasks:
+     - **Stake ARCH**: Stake ARCH tokens to earn extra gas credits.
+     - **Hold Drop Camp 'Astronomy' Patch NFT**: Possessing this NFT increases your free transaction quota.
+     - **Hold a Revealed Archie NFT**: Holding this NFT also boosts your allocation.
+
+3. **Broadcast Transactions Without ARCH Tokens**:
+   - Users can perform transactions such as minting an NFT on Ambur or swapping USDC to AXV without needing any ARCH tokens in their wallet.
+   
+## Dapp frontend integration
+
+We've added a basic [fee granter implementation guide](/developers/guides/cw-fees/introduction#frontend-interaction) that shows how you can use `arch3.js` to execute a transaction where the feegranter pays the gas fees. This is exactly what a frontend would need to do to utilize the Gas Guzzeler solution.
+
+The the Guzzler Club `mainnet` contract address is `archway1avwvqzu9gv86g5fxx5p2xqe0w33wklt27jusdrhszwccnfnxx0rsmzz8nu` and this would be used as the fee granter address.
+
+In order for a user's account to take advantage of the Gas Guzzler fee grant, their account would have to be whitelisted by the fee granter contract address above. This happens when an account accesses the [Guzzler Club](https://connect.archway.io/guzzler-club) and perform the required tasks.
```

Comment:
> This happens when an account accesses the Guzzler Club and perform the required tasks.

Not 100% true, you don't really have to perform actions in order to get into the whitelist. It's enough to sign in on ArchwayConnect and link your twitter account, and you will be added, claiming 3 ARCH of the default allowance automatically.

And yeah, in order to increase that allowance you would need to perform actions. E.g. stake, hold or just re-claim the default allowance in 21 days.